lua/git-worktree/init.lua

```diff
@@ -14,7 +14,7 @@ local function on_tree_change_handler(op, path, _) -- _ = upstream
         if op == Enum.Operations.Switch then
             local changed = M.update_current_buffer()
             if not changed then
-                vim.cmd(string.format(":Ex %s", M.get_worktree_path(path)))
+                vim.cmd(string.format("e %s", M.get_worktree_path(path)))
```

Comment:
I think we can use UV to see if this is a "real" path.  

That way we can selectively use :Ex (or whatever file tree they have) / open the file.  We can take care of that later.

Reply:
I made this change because Ex was failing for my headless nvim tests. 

It seemed to do the same actions. Is there a difference?

Reply:
no, there really isn't a difference.  Great point.